repository_service_tuf/helpers/api_client.py

```diff
@@ -69,6 +74,94 @@ def is_logged(server: str, token: str):
             return Login(state=True, data=data)
 
     else:
-        click.ClickException(
-            f"Error {response.status_code} {response.json()['detail']}"
+        raise click.ClickException(
+            f"Error {response.status_code} {response.text}"
+        )
+
+
+def get_headers(settings: Dict[str, str]) -> Dict[str, str]:
+    server = settings.get("SERVER")
+    token = settings.get("TOKEN")
+    if server and token:
+        token_access_check = is_logged(server, token)
+        if token_access_check.state is False:
+            raise click.ClickException(
+                f"{str(token_access_check.data)}"
+                "\n\nTry re-login: 'rstuf admin login'"
+            )
+
+        expired_admin = token_access_check.data.get("expired")
+        if expired_admin is True:
+            raise click.ClickException(
+                "The token has expired. Run 'rstuf admin login'"
+            )
+        else:
+            headers = {"Authorization": f"Bearer {token}"}
+            response = request_server(
+                server, URL.bootstrap.value, Methods.get, headers=headers
+            )
+            if response.status_code != 200:
+                raise click.ClickException(
+                    f"Unexpected error: {response.text}"
+                )
+    else:
+        raise click.ClickException("Login first. Run 'rstuf admin login'")
+
+    return headers
+
+
+def task_status(
+    task_id: str, server: str, headers: Dict[str, str], title: Optional[str]
+) -> Dict[Any, str]:
```

Comment:
nit: I think the type is `Dict[str, Any]`, taking into account https://github.com/vmware/repository-service-tuf-api/blob/main/repository_service_tuf_api/tasks.py#L21